visualizer/examples.js

```diff
@@ -162,6 +226,12 @@
         setExample(selectedId, cm.getValue());
         saveExamples();
       }
+    },
+    'Alt-S': function(cm) { // save (windows)
```

Comment:
It did last time I tried it on my linux box. But, I never added an 'e.preventDefault()' call.